final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,107 @@
+import feedparser
+#from arg import parsargs, VERSION
+from arg import parsargs, VERSION
+import clean_output
+from bs4 import BeautifulSoup 
+import json
+from loggs import logg, logg_json
+import logging
+import sys
+from datetime import datetime
+from add_to_csv import addcsv, out
+from converter import convert_date
+
+def get_sourse(parsed):
+    ''' Gets source information '''
+    feed = parsed['feed']
+    return {
+        'link': feed['link'],
+        'title': feed['title'],
+        'subtitle': feed['subtitle'],
+    }
+
+def get_news(parsed, console_args):
+    """ Gets entries information """
+    articles = []
+    entries = parsed['entries']
+    if console_args.limit is not None:
+        ''' Get right amount from the array '''
+        entries = entries[:console_args.limit]
+    for entry in entries:
+        img = BeautifulSoup(entry.summary, features="html.parser")
+        summary = BeautifulSoup(entry.summary, features='html.parser').text
+        article_img = img.find('img')['src']
+        articles.append({
+            'link': entry['link'],
+            'title': entry['title'],
+            'img': article_img,
+            'summary': summary,
+            'published': entry['published'],
+        })
+    return articles
+
+def output(article):
+    print("Title: ", 
+        clean_output.delete_unnecessary_symbols(article['title']))  #clean_output.print("Date: ", article['published'])
+    print("Date: ", article['published'])
+    print("Link: ", article['link'])
+    print("\nSummary: ", article['summary'])
+    print("\nImage: ", article['img'])
+    print('\n')
+
+def test_to_add(news_csv, articles):
+    for i in articles:
+        var = True
+        for j in news_csv:
+            if i['published'] != j[4]:
+                var = True
+            else:
+                var = False
+                break
+        if var:
+            addcsv(i)
+
+def main():
+    console_args = arg.parsargs()
```

Comment:
If you import specific function using `from` then you should just use it as is: `parsargs`, without `arg`